### PR TITLE
Important updates to SourcePanel User Interface

### PR DESCRIPTION
**Pull request by case141**
Removed placeholder text and replaced it to show Source Details when a Source Card is clicked.
- Renamed BrowserPanel to SourcePanel
- Added html generator in SourcePanel to generate source details

image

Partially Implements #74

### CODE REVIEW

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 1]
```diff
@@ -31,7 +31,7 @@
     private Logic logic;
 
     // Independent Ui parts residing in this Ui container
-    private BrowserPanel browserPanel;
+    private SourcePanel sourcePanel;
```
Comment (fterh) [inline src/main/java/seedu/address/ui/MainWindow.java:34]: Why did you rename browserpanel to sourcepanel?

Comment (case141) [reply src/main/java/seedu/address/ui/MainWindow.java:34]: BrowserPanel is the old name used by AddressBook4 and since our panel shows sources we should name it SourcePanel

**Review by fterh — APPROVED** [review]
